### PR TITLE
S04 us84 beliefs page

### PR DESCRIPTION
**Pull request by OCR-VH-DevWeb**
Version completed but need more work

### CODE REVIEW

**Review by Kileana27 — APPROVED** [review]

File: src/app/pages/beliefs-page/beliefs-page.component.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<section class="beliefs">
+  <h1 class="main-title">Les Croyances</h1>
+  @for(belief of beliefs; track belief){
```
Comment (Benja-san) [inline src/app/pages/beliefs-page/beliefs-page.component.html:3]: wrap this loop into ul tag

File: src/app/pages/beliefs-page/beliefs-page.component.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  <div class="beliefs-description-container">
```
Comment (Benja-san) [inline src/app/pages/beliefs-page/beliefs-page.component.html:4]: and wrap each belief into an li tag

File: src/app/pages/beliefs-page/beliefs-page.component.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  </div>
+  <div class="content-description">
+    <p>{{ belief.description }}</p>
+  </div>
+  <h3 class="intro-divinities bold-title">Les Divinités Du Royaume</h3>
+
+  <ul class="divinities-description-container">
+    @for(divinity of belief.divinities; track divinity){
+    <li class="style-li">
+      <h4 class="divinities-title bold-title">
```
Comment (Benja-san) [inline src/app/pages/beliefs-page/beliefs-page.component.html:15]: where is the h3 ?

File: src/app/pages/race/race.component.ts [hunk 4]
```diff
@@ -10,7 +10,7 @@ import { Race } from '../../model/race/race';
   styleUrl: './race.component.scss',
 })
 export class RaceComponent {
-  //déclare une propriété 'aces' de type array objects 'Race'
+  //déclare une propriété 'races' de type array objects 'Race'
```
Comment (Benja-san) [inline src/app/pages/race/race.component.ts:13]: remove dead code

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/pages/race/race.component.ts (hunk 4 above)
Comment (OCR-VH-DevWeb) [reply src/app/pages/race/race.component.ts:13]: This is not a dead code, it's a commentary

File: src/app/pages/beliefs-page/beliefs-page.component.html (hunk 3 above)
Comment (OCR-VH-DevWeb) [reply src/app/pages/beliefs-page/beliefs-page.component.html:15]: H3 is at line 10 ^^ haha

**Review by Benja-san — APPROVED** [review]

**Pull request merged**